runtime/src/main/java/org/corfudb/protocols/wireprotocol/LogData.java

```diff
@@ -81,6 +82,8 @@ public Object getPayload(CorfuRuntime runtime) {
                             } catch (Throwable throwable) {
                                 log.error("Exception caught at address {}, {}, {}",
                                         getGlobalAddress(), getStreams(), getType());
+                                log.error("Raw data buffer {}",
```

Comment:
I think you need to rewind the copyBuf reader's index, otherwise it might only print the part of the buffer that's unread.

Reply:
oh thanks!